specification.markdown

```diff
@@ -811,7 +811,9 @@ DocType string values of "matroska" and "webm" are reserved for pre-existing for
 
 This document creates a new IANA Registry called "CELLAR EBML Element ID Registry".
 
-Element IDs are described in section `Element ID`.  Element IDs are encoded using the VINT mechanism described in section (#variable-size-integer) can be between one and five octets long. Five octet long Element IDs are possible only if declared in the header.
+Element IDs are described in section `Element ID`. Element IDs are encoded using the VINT mechanism described in section (#variable-size-integer) can be between one and five octets long. Five octet long Element IDs are possible only if declared in the header.
+
+This IANA Registry only applies to `Elements` contained in the `EBML Header`, not the `Elements` used in the `EBML Body` which have their own set of independent `Element IDs`.
```

Comment:
Should this registry apply to Global Elements as well (Void and CRC-32)?

Reply:
Indeed, we need to explain what happens to global elements.